pkgs/os-specific/darwin/mysides/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl, libarchive, p7zip }:
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "mysides";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/mosen/mysides/releases/download/v${version}/mysides-${version}.pkg";
+    sha256 = "sha256-dpRrj3xb9xQSXXXxragUDgNPBaniiMc6evRF12wqVRQ=";
+  };
+
+  dontBuild = true;
+  nativeBuildInputs = [ libarchive p7zip ];
+
+  unpackPhase = ''
+    7z x $src
+    bsdtar -xf Payload~
+  '';
+
+  installPhase = ''
+    mkdir -pv $out/bin
```

```suggestion
    mkdir -p $out/bin
```
Verbose changes nothing here